Source/WebCore/animation/KeyframeEffect.cpp

```diff
@@ -107,7 +107,7 @@ static inline CSSPropertyID IDLAttributeNameToAnimationPropertyName(const String
         return CSSPropertyOffset;
 
     // 4. Otherwise, return the result of applying the IDL attribute to CSS property algorithm [CSSOM] to attribute.
-    auto cssPropertyId = CSSStyleDeclaration::getCSSPropertyIDFromJavaScriptPropertyName(idlAttributeName);
+    auto cssPropertyId = CSSStyleDeclaration::getCSSPropertyIDFromJavaScriptPropertyName(AtomString { idlAttributeName });
```

Comment:
Can we go even further back and make `idlAttributeName` be an `AtomString`?

Reply:
Technically, it comes from a JSC::Identifier, which, as far as I know is always backed by an AtomString. However, JSC::Identifier::string() returns a String instead of an AtomString for some reason.

Reply:
That’s too bad. Would be nice to clean up later.